packages/prettier-plugin-java/src/options.js

```diff
@@ -0,0 +1,239 @@
+"use strict";
+
+module.exports = {
+  entrypoint: {
+    type: "choice",
+    category: "Global",
+    default: "compilationUnit",
+    // sed -nr 's/.*\.RULE\(([^,]+),.*/\1/p' $(ls path/to/java-parser/rules/folder)
+    choices: [
```

Comment:
Is it possible to generate this file on release ? Or to add a test to check/commit-hook that check that every RULE has its description here ?

Reply:
I am not sure if we should generate this file on release.
I prefer to check with a hook, I have added one.